.github/workflows/weblate-update-pot.yml

```diff
@@ -0,0 +1,88 @@
+name: Weblate Update POT
+
+on:
+  schedule:
+    # run every working day (Monday-Friday) at 1:42AM UTC
+    - cron: "42 1 * * 0-4"
+
+  # allow running manually
+  workflow_dispatch:
+
+jobs:
+  update-pot:
+    # do not run in forks
+    if: github.repository == 'openSUSE/agama'
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: registry.opensuse.org/opensuse/tumbleweed:latest
+
+    steps:
+      - name: Configure and refresh repositories
+        # disable unused repositories to have a faster refresh
+        run: zypper modifyrepo -d repo-non-oss repo-openh264 repo-update && zypper ref
+
+      - name: Install tools
+        run: zypper --non-interactive install --no-recommends diffutils git gettext-tools
+
+      - name: Checkout Agama sources
+        uses: actions/checkout@v3
+        with:
+          path: agama
+
+      - name: Generate POT file
+        # TODO: use a shared script for this
+        run: |
+          cd agama/web
```

Comment:
so if I get it, it generate pot file only from web part and not from service ( aka ruby code ) and from rust code?

Reply:
Yes, web frontend only. As noted in the TODO above, this should be replaced by some other script.